release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,46 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+- Due to Azure limitations, old clusters can not be upgraded to support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
```

Comment:
This is where we'll add @T-Kukawka  migration recommendation for the customers.

Phrase "old clusters can not be upgrade" although it says later that just to use multi-az sounds a bit scary maybe we could rephrase? no ideas how yet 😁 good that you mentioned that it's due to azure limitations!

Reply:
I agree, I can imagine the heart of a customer jumping when reading that 😆 I'll think of better wording